### DISCUSSION TITLE
Dynamic Button CSS Classes

### DISCUSSION THREAD

**Discussion by ldelossa** [ErikReider/SwayNotificationCenter#280] [Q&A] [answered]
Hey there,

Is it possible to change the CSS/Icon of a button placed in the control center based on some external script. 

For instance, if a script determines wifi is disconnected, changing the CSS class to one which is red would be nice.

**Comment by ErikReider** [comment, marked as answer]
At the time of writing, these button are very simple and only run the script provided on click. A hacky way of doing this is to update the config JSON/CSS and hot reload to apply the changes?

**Reply by ldelossa** [reply]
Not a bad idea, can play with that but maybe a little clumsy for things like wifi

**Reply by ErikReider** [reply]
Maybe a stylable toggle button would be a good addition? I've actually planed on adding similar widgets to swaync in the future so this might not be necessary :)

**Reply by ldelossa** [reply]
A toggle with a state on and off could be simple and powerful 👏👍
